CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+- Changed: Refactor md-collection-loader
+  - Pluggable markdown parser (can be ascii parser or something else)
+  - Move loader query to webpack option (avoid json.stringify)
```

Comment:
We will need to review this message after switching to remark and providing a better default parser :)